builtin/files/plugins/dashboard/manifests/deployment.yaml

```
@@ -110,7 +110,7 @@ spec:
 
 {{ if .Values.ingress.enabled -}}
 ---
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: extensions/v1beta1
```

shouldn't be this the other way around? since extensions/v1beta1 is deprecated in 1.16 for ingress
```suggestion
apiVersion: networking.k8s.io/v1beta1
```

---

Hi, you are quite right!  I'm not sure how I managed to do that!  Great spot - thanks!